openpype/hosts/maya/api/pipeline.py

```diff
@@ -548,9 +548,10 @@ def on_task_changed():
 
 
 def before_workfile_save(event):
+    project_name = os.getenv("AVALON_PROJECT")
```

Comment:
_Just a question - I'm not sure what the best approach would be._

Should this maybe be `legacy_io.current_project()` or `legacy_io.active_project()` or is taking it from the environment the best way to go?

Looking over the code base `legacy_io.current_project` is only used twice. So likely it's not that one.
Other than that I see quite a mixture used between `os.getenv("AVALON_PROJECT")` (or `os.environ.get("AVALON_PROJECT")` and `legacy_io.Session["AVALON_PROJECT"]`.

Should these maybe get refactored at some point to what is the preferred entry point for the API to get the current project?

Reply:
Thanks - in that case this should be replaced with `legacy_io.active_project()`

Reply:
good point. @iLLiCiTiT is probably the right one to take a look . . .